src/PKSim.Core/Services/PKSimParameterStartValuesCreator.cs

```diff
@@ -1,56 +1,39 @@
 using OSPSuite.Core.Domain.Builder;
-using OSPSuite.Core.Domain.Formulas;
 using OSPSuite.Core.Domain.Services;
 using OSPSuite.Utility.Extensions;
 using PKSim.Core.Model;
-using static PKSim.Core.CoreConstants.CalculationMethod;
-using IFormulaFactory = PKSim.Core.Model.IFormulaFactory;
 
 namespace PKSim.Core.Services
 {
    public interface IPKSimParameterStartValuesCreator
    {
-      IParameterStartValuesBuildingBlock CreateFor(IBuildConfiguration buildConfiguration, Simulation simulation);
+      ParameterStartValuesBuildingBlock CreateFor(IBuildConfiguration buildConfiguration, Simulation simulation);
    }
 
    public class PKSimParameterStartValuesCreator : IPKSimParameterStartValuesCreator
    {
       private readonly IParameterStartValuesCreator _parameterStartValuesCreator;
-      private readonly IFormulaFactory _formulaFactory;
+      private readonly IObjectBaseFactory _objectBaseFactory;
       private readonly IEntityPathResolver _entityPathResolver;
-      private IParameterStartValuesBuildingBlock _defaultStartValues;
+      private ParameterStartValuesBuildingBlock _defaultStartValues;
 
-      public PKSimParameterStartValuesCreator(IParameterStartValuesCreator parameterStartValuesCreator,
-         IFormulaFactory formulaFactory,
+      public PKSimParameterStartValuesCreator(
+         IParameterStartValuesCreator parameterStartValuesCreator,
+         IObjectBaseFactory objectBaseFactory,
          IEntityPathResolver entityPathResolver)
       {
          _parameterStartValuesCreator = parameterStartValuesCreator;
-         _formulaFactory = formulaFactory;
+         _objectBaseFactory = objectBaseFactory;
          _entityPathResolver = entityPathResolver;
       }
 
-      public IParameterStartValuesBuildingBlock CreateFor(IBuildConfiguration buildConfiguration, Simulation simulation)
+      public ParameterStartValuesBuildingBlock CreateFor(IBuildConfiguration buildConfiguration, Simulation simulation)
       {
-         try
-         {
-            //default default parameter start values matrix
-            var spatialStructure = buildConfiguration.SpatialStructure;
-            var molecules = buildConfiguration.Molecules;
-            _defaultStartValues = _parameterStartValuesCreator.CreateFrom(spatialStructure, molecules);
-            var individual = simulation.Individual;
-
-            //set the relative expression values for each molecule defined in individual
-            individual.AllMolecules().Each(molecule => updateMoleculeParametersValues(molecule, individual, simulation));
-
-            updateSimulationParameters(simulation);
-
-            return _defaultStartValues.WithName(simulation.Name);
-         }
-         finally
-         {
-            _defaultStartValues = null;
-         }
+         //default default parameter start values matrix
+         _defaultStartValues = _objectBaseFactory.Create<ParameterStartValuesBuildingBlock>();
+         updateSimulationParameters(simulation);
```

Comment:
@PavelBal @Yuri05 Those are now the only parameters updated. In case of a brand new simulation , this will be empty